static/js/talks.js

```diff
@@ -0,0 +1,63 @@
+data = {
+    "talks": [
+        {
+            "title": "Olá Mundo",
+            "datetime": "2019-12-02T16:40:00.000",
+            "duration": 14400,
+            "embed_link": "https://www.youtube.com/embed/3bIw6294Skk",
+            "link": "https://www.youtube.com/watch?v=3bIw6294Skk"
+        },
+        {
+            "title": "Olá Outro Mundo",
+            "datetime": "2019-12-02T16:30:15.000",
+            "duration": 14400,
+            "embed_link": "https://www.youtube.com/embed/3bIw6294Skk",
+            "link": "https://www.youtube.com/watch?v=3bIw6294Skk"
+        },
+    ]
+}
+
+function getTimeZonedDate(offset) {
+    // create Date object for current location
+    var d = new Date();
+
+    // convert to msec
+    // subtract local time zone offset
+    // get UTC time in msec
+    var utc = d.getTime() + (d.getTimezoneOffset() * 60000);
+
+    // create new Date object for different city
+    // using supplied offset
+    var nd = new Date(utc + (3600000*offset));
+
+    return nd
+}
+
+function isHappeningNow(talk) {
+  //const now = new Date()
+  const now = getTimeZonedDate('-2')
```

Comment:
Sem horário de verão, estamos no -3
```suggestion
  const now = getTimeZonedDate('-3')
```

Reply:
> Sem horário de verão, estamos no -3

Para mim só funciona se eu passar `-1` aqui 🤔